test/e2e/performanceprofile/functests/utils/consts.go

```diff
@@ -101,6 +101,8 @@ const (
 	NamespaceMachineConfigOperator = "openshift-machine-config-operator"
 	// NamespaceTesting contains the name of the testing namespace
 	NamespaceTesting = "performance-addon-operators-testing"
+	// DeploymentTesting contains the name of the deployment namespace
+	DeploymentTesting = "test-deployment"
```

Comment:
Can't we use the `NamespaceTesting`?

Reply:
Theoretically we can. I though we could use a name for the deployment that is more accurate rather than reusing the namespace of the deployment to its name.

Reply:
We have a logic to delete the everything under the `NamespaceTesting` at the end of the test so it save us the burden of making sure we delete the pods/deployments. 
Having another namespace that we should take care of and do the same, is a bit tedious in my opinion.

Reply:
but it is not another namespace. so that wouldn't be a problem right?

Reply:
The comment on line 104 says it is

Reply:
Correct it.